Assets/MRTK/Providers/Experimental/WindowsSceneUnderstanding/WindowsSceneUnderstandingObserver.cs

```diff
@@ -760,10 +773,21 @@ private async Task RunObserverAsync(CancellationToken cancellationToken)
         /// Gets the matrix representing the transform from scene space to world space
         /// </summary>
         /// <returns>The transform matrix</returns>
-        private System.Numerics.Matrix4x4 GetSceneToWorldTransform()
+        private System.Numerics.Matrix4x4? GetSceneToWorldTransform()
         {
             var result = System.Numerics.Matrix4x4.Identity;
 #if WINDOWS_UWP
+#if MSFT_OPENXR
```

Comment:
I wonder if we also want to check that the active loader is OpenXR? For example, even though I have the Mixed Reality OpenXR Plugin package installed, I might still want to run with Windows XR Plugin to test or something.